From 3531ff73631a0d59234eb4713e7b3a7f5ea57bbb Mon Sep 17 00:00:00 2001
From: Nicola Lunghi <nicola.lunghi@jci.com>
Date: Thu, 14 Nov 2019 12:17:51 +0000
Subject: [PATCH] setup.py: move pytest-runner to test_requirements

This fixes an issue with yocto build.
pytest-runner is only needed when running tests.

Upstream-Status: Pending
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 53ebea7..ebb0de2 100644
--- a/setup.py
+++ b/setup.py
@@ -10,9 +10,9 @@ with open('README.md') as readme_file:
 
 requirements = []
 
-setup_requirements = ['pytest-runner', ]
+setup_requirements = []
 
-test_requirements = ['pytest>=3', ]
+test_requirements = ['pytest>=3', 'pytest-runner']
 
 setup(
     author="Nicolas Aimetti",
-- 
2.20.1

